packages/canary-web-components/src/components/ic-card-horizontal/ic-card-horizontal.css

```diff
@@ -0,0 +1,207 @@
+@import "../../../dist/core/normalize.css";
+
+:host {
+  display: flex;
+
```

Comment:
It would be good to be able to choose the width of the card using a custom CSS variable